operations/node_management/Change_Settings_for_HMS_Collector_Polling_of_Air_Cooled_Nodes.md

```diff
@@ -20,15 +20,6 @@ information returned from the BMCs.
 
```

Comment:
There are two references to CAPMC in the paragraphs above.

Reply:
Ref OK, having an example bad.